.github/workflows/post-pr-comment.yml

```diff
@@ -0,0 +1,31 @@
+name: Post pull-request comment
+on:
+  workflow_run:
+    workflows: ["Query help preview"]
+    types:
+      - completed
+
+permissions:
+  pull-requests: write
+
+jobs:
+  post_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@af92a8455a59214b7b932932f2662fdefbd78126
```

Comment:
I think you could get rid of this external action by doing something like this:
`gh run download --name "comment" "$RUN_ID"`

Reply:
Thanks that is much simpler; in addition we don't need to worry about the safety of the external action.